finder-app/finder.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+# Check for number of arguments
+if [ ! $# -eq 2 ]
+then
+	echo "Invalid number of arguments! Needs to be 2"
```

Comment:
@suryakanteti think about how you could make this statement more clear for the user to describe what they need to do.